### PR TITLE
10.1.0.Final

### PR DESCRIPTION
**Pull request by tristantarrant**

### CODE REVIEW

File: blog/2019-12-20-infinispan-10.1.0.Final.adoc [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+= Infinispan 10.1 "Turia"
+
+Hi there,
+
+we finish 2019 in style with the Final release of Infinispan 10.1, codenamed https://www.cervezaturia.es["Turia"].
+
+== Server console
+
+The highlight of this release is the new server console which is now based on https://www.patternfly.org/v4/[Patterfly 4] and https://www.patternfly.org/v4/[React.js].
```
Comment (pruivo) [inline blog/2019-12-20-infinispan-10.1.0.Final.adoc]: the link for Reac.js is pointing to patternfly webapage.
shouldn't be https://reactjs.org/ ?

File: blog/2019-12-20-infinispan-10.1.0.Final.adoc [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+[caption="Console: cache stats",link=/blog/img/ispn101console2.png]
+image::/blog/thumb/ispn101console2.png[Console Cache Stats]
+
+== Security
+
+Many changes related to security have happened since 10.0:
+
+* Native SSL/TLS provided by https://github.com/wildfly-security/wildfly-openssl[WildFly OpenSSL]. The server only ships with native libraries for Linux x86_64, but you can download natives for other platforms and architectures
+* Improved usability of client configuration with better defaults
```
Comment (pruivo) [inline blog/2019-12-20-infinispan-10.1.0.Final.adoc]: replace `client` with 'Hot Rod client`?

File: blog/2019-12-20-infinispan-10.1.0.Final.adoc [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+* A command-line switch to specify an alternate logging configuration file
+* Query and indexing operations/stats are now exposed over the RESTful API
+* Tasks and Scripting support
+* Support for binding the endpoints to 0.0.0.0 / ::0 (aka INADDR_ANY)
+
+== Non-blocking
+
+More work has landed on the quest to completely remove blocking calls from our internals. The following have been made non-blocking:
+
+* state transfer
```
Comment (pruivo) [inline blog/2019-12-20-infinispan-10.1.0.Final.adoc]: start the sentence with a capital letter? 
same for below

File: blog/2019-12-20-infinispan-10.1.0.Final.adoc [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+* The Infinispan Lucene Directory has been deprecated.
+* The memcached protocol server has been deprecated. If you were relying on this, come and talk to us about working on a binary protocol implementation.
+
+== Bug fixes, clean-ups and documentation
+
+Over 120 issues fixed including a lot of documentation updates. See the link:[full list of changes and fixes]
+
+== Get it, Use it, Ask us!
+
+Please http://infinispan.org/download/[download],
```
Comment (pruivo) [inline blog/2019-12-20-infinispan-10.1.0.Final.adoc]: https?

File: blog/2019-12-20-infinispan-10.1.0.Final.adoc (hunk 1 above)
Comment (tristantarrant) [reply blog/2019-12-20-infinispan-10.1.0.Final.adoc]: Fixed

File: blog/2019-12-20-infinispan-10.1.0.Final.adoc (hunk 2 above)
Comment (tristantarrant) [reply blog/2019-12-20-infinispan-10.1.0.Final.adoc]: yep

File: blog/2019-12-20-infinispan-10.1.0.Final.adoc (hunk 4 above)
Comment (tristantarrant) [reply blog/2019-12-20-infinispan-10.1.0.Final.adoc]: Well spotted

File: blog/2019-12-20-infinispan-10.1.0.Final.adoc (hunk 3 above)
Comment (tristantarrant) [reply blog/2019-12-20-infinispan-10.1.0.Final.adoc]: Done

**Pull request merged**